apps/ponder-interop/src/createPonderConfig.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  contracts as addrs,
+  l2ToL2CrossDomainMessengerAbi,
+} from '@eth-optimism/viem'
+import { createConfig } from 'ponder'
+import type { Transport } from 'viem'
+
+export type Endpoint = { chainId: number; transport: Transport }
+
+/**
+ * A map of name to {@link Endpoint}
+ */
+export type Endpoints = Record<string, Endpoint>
+
+/**
+ * Create a Ponder config for interop
+ * @param endpoints - Interoperable endpoints to index -- {@link Endpoints}
+ * @returns Ponder configuration
+ */
+export function createPonderConfig(endpoints: Endpoints) {
+  if (Object.keys(endpoints).length === 0) {
+    throw new Error('no endpoints provided')
+  }
+
+  // relevant interop contracts
+  const contracts = {
+    L2ToL2CDM: {
+      abi: l2ToL2CrossDomainMessengerAbi,
+      startBlock: 1,
```

Comment:
side question - will it be possible to modify this on a per chain basis. it should ideally be equally to the block where interop first started on the chain, right? everything will still work if it is set to 1, but might be nice for performance if it can be set to a later value.

Reply:
ahh yea that's right. When we get to launch, we can make this configurable as a parameter